modules/product/src/main/java/com/opengamma/strata/product/bond/BillYieldConvention.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2018 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product.bond;
+
+import org.joda.convert.FromString;
+import org.joda.convert.ToString;
+
+import com.opengamma.strata.collect.named.EnumNames;
+import com.opengamma.strata.collect.named.NamedEnum;
+
+/**
+ * A convention defining how yield is computed for a bill.
+ */
+public enum BillYieldConvention implements NamedEnum {
+  
+  /**
+   * Discount.
+   */
+  DISCOUNT("Discount"),
+  
+  /**
+   * France CD: interest at maturity.
+   */
+  FRANCE_CD("France-CD"),
+  
+  /**
+   * Interest at maturity.
+   */
+  INTEREST_AT_MATURITY("Interest-at-maturity"),
```

Comment:
Are these names based on something standard? The bond conventions have the constant name match the string name closely. I'd expect this to be `Interest-At-Maturity`